cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -1,5 +1,41 @@
 <header class="o-header"
         role="banner">
+
+    {% if flag_enabled('BETA_NOTICE') %}
```

Comment:
it seems like it might choke when being inside an include (that's why my demo broke when at the D+D presentation), but this doesn't seem to have triggered that issue.